README.md

```diff
@@ -37,6 +37,44 @@ You can configure default value of analyze settings like example below.
 ```jsonc
 "WavPreview.analyzeDefault": {
         // Settings about WaveForm
+        /*
+         Make the waveform visible or hidden.
+         true: visible, false: hidden
+         default: true
+        */
+        "waveformVisible": true,
+
+        /*
+         Adjust height of the waveform. 
+         The valid range of [0.3, 2.0] scales the default height.
+         default: 1.0
+         This option can only be configured through the settings file.
+        */
+        "waveformVerticalScale": 1.0,
+
+        /*
+         Make the waveform channel label shown or hidden.
+         true: shown, false: hidden
+         default: false
+        */
+        "waveformShowChannelLabel": false,
+
+        /*
+         Enable nicely rounded waveform axis.
+         true: rounded, false: just divided the height
+         default: true
+         This option can only be configured through the settings file.
+        */
+        "roundWaveformAxis": true,
```

Comment:
Rounding the axis values is certainly easier to see. I want to always use the code you create.
I apologize for the inconvenience, but is it possible to delete the original axis implementation? You don't even need to be able to change it in the settings. The same goes for timeAxis.